### PR TITLE
Adding alert functionality to cloud-iot-rtdp tutorial

### PR DESCRIPTION
**Pull request by kingman**
@gguuss @ptone  
Hi Gus,
Apologies for adding these last minutes changes to this tutorial, I been adding some more content to this tutorial that I really should communicated with you earlier. Really sorry for adding this extra work on you. Here is a summary of what I have done:

- Add script for storing temperature threshold value for each device in Datastore
- Fetch threshold value, doing temperature check in and error logging on exceeding value from cloud function
- Add more business background narrative for the setup

I'm really grateful if you could do a quick review of the changes made and merge these into the community repo.

Best,
Charlie
 
- [x] Subject matter expert review - @gguuss 
- [x] Formatting review for publishing - @jmdobry

### CODE REVIEW

**Comment by kingman** [comment]
I have checked the CLA, and have already a corporate signer.

File: tutorials/cloud-iot-rtdp/bin/create_temp_alert_store.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import os
+from google.cloud import datastore
+
+client = datastore.Client(os.environ['PROJECT'])
```
Comment (gguuss) [inline tutorials/cloud-iot-rtdp/bin/create_temp_alert_store.py]: We usually use GCLOUD_PROJECT or GOOGLE_PROJECT_ID for this environment variable.

File: tutorials/cloud-iot-rtdp/bin/create_temp_alert_store.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+for i in range(30):
+    incomplete_key = client.key('device')
+    device = datastore.Entity(key=incomplete_key)
+    device.update({
+        'name':u'device'+str(i+1),
```
Comment (gguuss) [inline tutorials/cloud-iot-rtdp/bin/create_temp_alert_store.py]: prefer:

    'name':u'device{}'.format(str(i+1))

File: tutorials/cloud-iot-rtdp/index.md [hunk 3]
```diff
@@ -180,10 +187,23 @@ Core to receive data from MQTT clients.
     Warning: This tutorial includes a public/private key pair for testing
     purposes. Do not use this pair in a production environment.
 
+## Create threshold values in DataStore
+In this section, you insert threshold values for each of the devices, registered in the IoT Core Device Manager, in the DataStore.
+1. In Cloud Shell, run python script to insert the device objects into Datastore:
+
+        sudo pip install google-cloud-datastore
```
Comment (gguuss) [inline tutorials/cloud-iot-rtdp/index.md]: I prefer to use virtualenv to running pip as root, e.g.:

    virtualenv env && source env/bin/activate
    pip install -r requirements.txt 
    ...

**Comment by kingman** [comment]
@gguuss 
I have now adjusted according to you comments and pinged Mike P for content review. After that we should be ready for merge.
/Charlie

**Comment by kingman** [comment]
@gguuss 
The changes done by Mike P is now merged. All changes are now in place for merge into master.

**Comment by gguuss** [comment]
Still LGTM, I don't have merge access. @jmdobry does some checks on the attributes / tags and should be able to merge when the article is ready to republish.

**Comment by jmdobry** [comment]
I'll take a look soon.

**Pull request merged**